app/controllers/users_controller.rb

```diff
@@ -15,4 +15,8 @@ def update
       render :edit
     end
   end
+
+  def bids_placed
```

Comment:
I am not line love with having non-RESTful routes like this. Why not map the same route we were using before (`bids#index`) to the path name that we want? As a rails dev, that routing makes more sense to me